services/web/project/service/metrics_service.py

```diff
@@ -0,0 +1,21 @@
+from ..repository import metrics_calculation_repository, metrics_repository
+from ..model import Metrics
+from multiprocessing import Pool
+
+
+def calculate_shared_repos():
+     query_result = metrics_calculation_repository.shared_repos()
+     __save_all(query_result)
+
+def __save_all(query_result): # TODO fazer insert em lote (bulk)
+    [__save(r) for r in query_result]
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expression "[__save(r) for r in query_result]" is assigned to nothing](https://app.codacy.com/manual/mpgabriel95/TCC/pullRequest?prid=5538946)